### PR TITLE
thermoModel subspace edits

### PR DESCRIPTION
**Pull request by rmtfleming**
*Please include a short description of enhancement here*

**I hereby confirm that I have:**

- [] Tested my code on my own machine
- [] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/analysis/exploration/printLabeledData.m [hunk 1]
```diff
@@ -17,7 +17,8 @@ function printLabeledData(labels, data, nonzeroFlag, sortCol, fileName, headerRo
 % .. Authors: Markus Herrgard 6/9/06
 
 tol = 1e-9;
-format = '%g\t';
+%format = '%g\t';
```
Comment (laurentheirendt) [inline src/analysis/exploration/printLabeledData.m]: We need to have a check here then, as this format fails on Windows (the `%g` is cross platform).

**Comment by tpfau** [comment]
Not sure whether the format change is good if the fid is not standard out. 
Currently the output is a valid tab separated value format, after the change, the first column no longer fits to this scheme and all ids will have spaces added when read in.
Maybe a format switch would be good here.
If fid is an actual file: output it as tsv without padding.
If fid is std out: output it with padding (potentially dependent on the maximal size of identifier column?)

Also: The header column should have the same format as the individual columns. Otherwise the header is not indented correctly.

**Comment by laurentheirendt** [comment]
@tpfau, may you branch off and commit your suggestion to see? Thanks.

**Comment by tpfau** [comment]
please have a look at my [pr1081](https://github.com/tpfau/cobratoolbox/tree/pr1081) branch

**Comment by laurentheirendt** [comment]
Great! 🎉

**Pull request merged**